app/assets/stylesheets/partials/wice_grid.scss

```diff
@@ -37,67 +32,61 @@
     }
 
     .collapse-multi-select-icon{
-      background: transparent url(/assets/icons/grid/collapse.gif) no-repeat;
+      background: transparent asset-url('/assets/icons/grid/collapse.gif') no-repeat;
     }
 
     .expand-multi-select-icon{
-      background: transparent url(/assets/icons/grid/expand.gif) no-repeat;
+      background: transparent asset-url('/assets/icons/grid/expand.gif') no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Prefer double-quoted strings